app/frontend/src/search/ui/stats.js

```diff
@@ -0,0 +1,41 @@
+export const renderStats = (renderOptions, isFirstRender) => {
+  const { nbHits, nbPages, page, hitsPerPage, widgetParams } = renderOptions;
+
+  if (isFirstRender) {
+    return;
+  }
+
+  let results = constructResults(nbHits);
+  let last = constructLastResult(nbPages, page, nbHits, hitsPerPage);
+  let first = constructFirstResult(nbPages, page, hitsPerPage);
+
+  widgetParams.container.innerHTML = `
+    Showing <b>${first}</b> to <b>${last}</b> of <b>${nbHits}</b> ${results}
```

Comment:
using the b tag is a bit old school and not considered best practise for styling purposes. better to use spans and apply the GDS style